app/views/capacity.html

```diff
@@ -40,6 +40,71 @@
 
 <div id="plotly-div-line" role="presentation"></div>
 
+{% if organisationLevel !== 'offender-manager' %}
+  <h3 class="heading-medium">Capacity breakdown</h3>
+  <table id="example" class="data-table dataTable">
+    <thead>
+      <tr>
+        <th colspan="2" id="team">{{ childOrganisationLevelDisplayText }}</th>
+        <th class="" colspan="14" id="breakdown">Capacity breakdown</th>
```

Comment:
Any need for an empty `class` attribute?

Reply:
Should `colspan` here not be 8 as there are only 10 columns and 2 of them are consumed by the preceding header.?

Reply:
More copy pasta, removed and ammended